code/modules/clothing/shoes/magboots.dm

```diff
@@ -10,13 +10,15 @@
 	species_fit = list(VOX_SHAPED)
 	footprint_type = /obj/effect/decal/cleanable/blood/tracks/footprints/magboots
 
-	var/stomp_attack_power = 20
+	var/stomp_attack_power = 45
 
 /obj/item/clothing/shoes/magboots/on_kick(mob/living/carbon/human/user, mob/living/victim)
 	if(!stomp_attack_power)
 		return
 
 	var/turf/T = get_turf(src)
+	var/datum/organ/external/affecting = victim.get_organ(user.get_unarmed_damage_zone(victim))
+	
 	if(magpulse && victim.lying && T == victim.loc && !istype(T, /turf/space)) //To stomp on somebody, you have to be on the same tile as them. You can't be in space, and they have to be lying
 		//NUCLEAR MAGBOOT STUMP INCOMING (it takes 3 seconds)
```

Comment:
Since there's no check to see if you're already stomping, I take it you can queue up five stomps to instantly render them uncloneable after 3 seconds resolves?